tests/integration/sync/sync_integ_base.py

```diff
@@ -262,6 +263,8 @@ def get_sync_command_list(
             command_list += ["--image-repository", str(image_repository)]
         if image_repositories:
             command_list += ["--image-repositories", str(image_repositories)]
+        if s3_bucket:
+            command_list += ["--s3-bucket", str(s3_bucket)]
```

Comment:
Since we added support for s3 bucket for integration test, we can also write a simple one with s3_bucket option using the package integration test's testing bucket.